cmd/output.go

```diff
@@ -0,0 +1,42 @@
+package cmd
+
+import (
+	"fmt"
+	"log/slog"
+
+	"github.com/spf13/cobra"
+
+	"github.com/fogo-sh/almanac/pkg/content"
+	"github.com/fogo-sh/almanac/pkg/templates"
+)
+
+var outputCmd = &cobra.Command{
+	Use:   "output",
+	Args:  cobra.NoArgs,
+	Short: "Output all pages to disk",
+	Run: func(cmd *cobra.Command, args []string) {
+		contentDir := must(cmd.Flags().GetString("content-dir"))
+
+		pages, err := content.DiscoverPages(contentDir)
+		if err != nil {
+			checkError(err, "failed to discover pages")
+		}
```

Comment:
```suggestion
		checkError(err, "failed to discover pages")
```
checkError removes the need for if err != nil